src/detections/hooks/use-init.js

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+
+import { actions } from 'detections/fetch/slices/get'
+
+export const useInit = () => {
```

Comment:
Let's rename this to `useGetDetections`.

Reply:
done